src/essentials/Colors/docs/SemanticColorsTable.tsx

```diff
@@ -32,7 +32,7 @@ export const SemanticColorsTable: FC = () => {
     const [nameSearchInput, setNameSearchInput] = useState('');
     const flatSemanticColors = flattenObj(SemanticColors);
 
-    const filteredColorKeys = Object.keys(flatSemanticColors).filter(it => {
+    const filteredColorKeys = ([...flatSemanticColors.keys()] as string[]).filter(it => {
```

Comment:
One liner alternative: `Array.from(mapObj.keys()).filter(it => !searchStr || it.includes(searchStr))`

Reply:
Makes a lot of sense, I've also moved the `flatSemanticColors` and the `.keys()` outside the component to not have to redeclare them on every render